config/config_test.go

```diff
@@ -405,6 +405,23 @@ func TestMinTLSVersion(t *testing.T) {
 	})
 }
 
+func TestExcludeInsecureCipherSuites(t *testing.T) {
+
+	t.Run("test default", func(t *testing.T) {
+		cfg := new(Config)
+		if cfg.filterCipherSuites() != nil {
+			t.Error("No cipher suites should be filtered by default")
+		}
+	})
+	t.Run("test excluded insecure cipher suites", func(t *testing.T) {
+		cfg := new(Config)
+		cfg.ExcludeInsecureCipherSuites = true
+		if cfg.filterCipherSuites() == nil {
+			t.Error(`Failed to get list of filtered cipher suites`)
+		}
+	})
+}
```

Comment:
I think its better to test that ToOptions produces a list with excluded insecure suites rather than test unexported method.